LanguageExploration.php

radewa opened the pull request:
Mohon dikoreksi pak

radewa committed:
Create LanguageExploration.php

Mohon dikoreksi pak

pascalalfadian commented:
Dear @radewa, Pastikan dulu tidak ada compile error:

```
C:\Users\pascal.FTIS.000\Desktop>c:\xampp\php\php.exe LanguageExploration.php

Parse error: syntax error, unexpected '$privateInt' (T_VARIABLE), expecting func
tion (T_FUNCTION) in C:\Users\pascal.FTIS.000\Desktop\LanguageExploration.php on
 line 9
```

radewa committed:
Update LanguageExploration.php

Sudah perbaiki... Tinggal ada compile error.

radewa commented:
Perbaikan sudah dilakukan, tidak ada compile error lagi. :-)

pascalalfadian commented:
Masih ada error, tapi tidak terlihat karena kamu tidak menjalankan fungsi `main()` nya.

Coba kamu tambahkan kode ini di baris 103...

``` php
        $le = new LanguagesExploration();
        $le->main();
```

... maka kamu akan melihat errornya

radewa committed:
Update LanguageExploration.php

Perbaikan menyeluruh, seharusnya sudah bisa dijalankan tanpa ada error sama sekali. :-)

radewa commented:
Sudah diperbaiki kembali Pak. Seharusnya tidak ada error ketika di compile dan bisa jalan sesuai permintaan. :-)

radewa committed:
Update LanguageExploration.php

radewa committed:
Update LanguageExploration.php

pak sudah dibetulkan dengan nama kelompoknya didalamnya

pascalalfadian commented:
Memang sudah tidak error, tetapi kamu menghilangkan poin 14-23. Karena sudah melewati batas pengumpulan, tidak perlu diperbaiki lagi. Terima kasih.
